scripts/ci/travis.sh

```diff
@@ -67,35 +67,24 @@ done
 
 ./cardano-sl-tools.root/bin/cardano-wallet-hs2purs
 
+# Generate daedalus-bridge
 pushd daedalus
   nix-shell --run "npm install && npm run build:prod"
   echo $TRAVIS_BUILD_NUMBER > build-id
   cp ../log-config-prod.yaml .
 popd
 
+# Generate explorer frontend
+EXPLORER_EXECUTABLE=$(pwd)/cardano-sl-explorer-static.root/bin/cardano-explorer-hs2purs ./explorer/frontend/scripts/build.sh
```

Comment:
What's wrong with a single line? :) Could also export in separate line if you'd like.

The reason for pwd is that the script maniuplates pwd, so it has to be absolute otherwise resolving relative path would be hard